bin/src/ctl/mod.rs

```diff
@@ -40,305 +37,278 @@ pub fn ctl(matches: cli::Sozu) -> Result<(), anyhow::Error> {
     let mut channel = create_channel(&config).with_context(|| {
         "could not connect to the command unix socket. Are you sure the proxy is up?"
     })?;
+
+    // make the channel blocking, everything in sozuctl is synchronous anyway
+    channel.set_blocking(true);
```

Comment:
Is this working? I saw an usage of this using raw file descriptor to really set the channel blocking

Reply:
A fine remark. It is working but not usefull. I'll remove it.